tests/tests/test-replica-sets.spec.js

```diff
@@ -56,7 +56,9 @@ context('Test Replica Sets', () => {
       .contains(`${REPLICA_SET_NAME}-`)
       .click();
 
-    cy.getIframeBody().contains(`Name${REPLICA_SET_NAME}`);
+    cy.getIframeBody().contains(REPLICA_SET_NAME);
```

Comment:
Why those changes?

Reply:
Because for now, we aren't displaying names in the table, only in the header of the container.